Updates to cloud getting started (#19657)
gravitational/teleport: docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

## Step 4/5 Access Server

Select close and Server can be accessed via SSH
![Access Server](../../../img/cloud/online-session-5.png)

Interact with your SSH session
![Open SSH](../../../img/cloud/online-ssh-5.png)

Exit the session with `exit` and you can replay the session within Session Recordings
![Replay Session](../../../img/cloud/online-replay-5.png)

## Step 5/5 Access from Command Line

Install client libraries:

<Tabs>

>   <TabItem label="MacOS">

Reviewer: This should probably be a partial.

Author: @zmb3 moved to partial and updated to use cdns

  You can download one of the following .pkg installers for macOS:

  |Link|Binaries|
  |-|-|

>   |[`teleport-(=teleport.version=).pkg`](https://get.gravitational.com/teleport-ent-(=teleport.version=).pkg)|`teleport`<br/>`tctl`<br/>`tsh`<br/>`tbot`|

Reviewer: Let's use cdn.teleport.dev going forward.

## Next Steps

> - Install and use [Teleport Connect](../connect-your-client/teleport-connect.mdx) as a Desktop Graphical Client to access your resources.

Reviewer: ```suggestion
- Consider using our desktop app - [Teleport Connect](../connect-your-client/teleport-connect.mdx) to access your resources.
```
